src/redux/reducers/genes/initialState.js

```diff
@@ -21,6 +21,7 @@ const initialState = {
     error: false,
     data: {},
     views: {},
+    expressionType: 'raw',
```

Comment:
shouldn't `expressionType` be under `expression` in the `genes` store? At the end of the day, it is a thing specific to the expression of the genes, the `properties` part of the `genes` store does not care about it.

Reply:
It is actually under `expression` (line 19), it's cut by the diff (have to expand)

Reply:
It's actually under `expression`. There property is there on line 19, but the diff cut it so have to expand to see it,